codespell_lib/data/dictionary.txt

```diff
@@ -1650,6 +1664,22 @@ ambuguity->ambiguity
 ambulence->ambulance
 ambulences->ambulances
 amdgput->amdgpu
+amealearate->ameliorate
+amealearative->ameliorative
+amealearator->ameliorator
+amealiarate->ameliorate
+amealiarative->ameliorative
+amealiarator->ameliorator
+ameba->amoeba
```

Comment:
This is an interesting one.  A couple of things:

1. "ameba" is an uncommon but correct alternate spelling of "amoeba."
2. The pluralization of ameba/amoeba is disputed; in microbiology, some sources state that the plural properly has the Latinate construction "amoebae"(1), although many common encyclopaedia and dictionaries(2) and some textbooks(3) allow "amebas" and "amebae." Wikipedia also cites "amœba."(4)
 
So, while the short/anglicized plural forms are "okay", mixing alternates in the same piece of writing is not.  E.g. one shouldn't use both "amemba" and "amoebas" in the same text, especially a piece of technical writing in which the author is concerned enough to use CodeSpell.

In short, I think these changes will result in consistent, most-correct usages:

Add amebas->amoebae
Add amebae->amoebae
Add amoebas->amoebae
Add amœba->amoeba
Add amœbae->amoebae
Add amœbas->amoebae

References:

(1) https://www.caister.com/highveld/microbiology/what-is-an-amoeba.html
(2) https://www.merriam-webster.com/dictionary/amoeba
(2) https://www.britannica.com/science/amoeba-order
(2) https://www.oxfordlearnersdictionaries.com/us/definition/english/ameba?q=amebas
(2) https://www.lexico.com/en/definition/amoeba
(3) https://www.ncbi.nlm.nih.gov/books/NBK7742/
(4) https://en.wikipedia.org/wiki/Amoeba